### PR TITLE
Add CONTRIBUTING.md template

### PR DESCRIPTION
**Pull request by bhack**
This is a Draft CONTRIBUTING.md template.
It is part of an effort to standardize some early access info for [Github community health files](https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file) across the TensorFlow ecosystem repositories.

### CODE REVIEW

File: governance/SIG-contributing-template.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## TESTS
+How to write and execute tests (bazel, pytest, etc.) or a link to a specific [TEST.md]
+
+## DEBUG
+How to debug your code
+
+## CI
+Quickly describe your CI
+
+## CODEOWNERS
```
Comment (bhack) [inline governance/SIG-contributing-template.md:42]: It  could be great if we could standardize and automatically [monitor the codeowners/mentors](https://github.com/tensorflow/addons/pull/2024) availability.

Comment (bhack) [reply governance/SIG-contributing-template.md:42]: We could take some idea from the [Addons proxy maintainership](https://github.com/tensorflow/community/blob/master/rfcs/20190308-addons-proxy-maintainership.md) also if some of these theoretic best practices are not applied.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by bhack** [comment]
I've reverted this in a new PR cause It was only a draft

**Pull request merged**